src/tests/Microsoft.PowerFx.Core.Tests/BindingEngineTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Linq;
+using Microsoft.PowerFx.Core;
+using Microsoft.PowerFx.Core.Public.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Tests
+{
+    public class BindingEngineTests
+    {
+        [Fact]
+        public void CheckSuccess()
```

Comment:
>CheckSuccess [](http://example.com/codeflow?start=20&length=12)

Allow tools to bind expressions with just FxCore (and not have to pull in interpreter).